server/src/api/resources/trip/trip.controller.js

```diff
@@ -137,3 +144,37 @@ export const repeatTrip = (req, res) => {
   delete updatedRequest.body.id
   createTrip(updatedRequest, res)
 }
+
+export const uploadPics = (req, res) => {
+  const id = req.params.id
+  cloudinary.v2.uploader.upload(req.body.image, (err, result) => {
+    Trip.findOneAndUpdate(
+      { _id: id },
+      { $push: { tripPics: result.url } },
+      { returnOriginal: false }
+    ).then(oldTrip => {
+      Trip.findOne({ _id: oldTrip.id })
+        .then(newTrip => {
+          res.status(200).json(newTrip)
+        })
+        .catch(() => {
+          res.status(404).json("Not Found")
+        })
+        .catch(error => {
+          console.log(error, "ERROR")
+          res.status(404).json(error)
+        })
+    })
+  })
+}
+
+export const renderPics = (req, res) => {
```

Comment:
What's this endpoint for? If the photos are already on the Trip object in an array then the photos should already be in Redux state, right?

Reply:
deleted this. Didnt need it!